com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/font/Fontc.java

```diff
@@ -231,17 +244,26 @@ public void TTFBuilder(InputStream fontStream) throws FontFormatException, IOExc
 
         if (!fontDesc.getAllChars()) {
 
-            // 7-bit ASCII. Note inclusive range [32,126]
-            for (int i = 32; i <= 126; ++i) {
-                characters.add(i);
+            String chars = fontDesc.getCharacters();
+            if (!StringUtils.isEmpty(chars)) {
+                for (int i = 0; i < chars.length(); i++) {
+                    char c = chars.charAt(i);
+                    characters.add((int) c);
+                }
             }
```

Comment:
logic that can read both Old and New path. If new chars field is filled, then use this one, else use old path